### DISCUSSION TITLE
Empresas com sites diferentes para propositos diferentes

### DISCUSSION THREAD

**Discussion by phhoffmann** [frontendbr/forum#1502] [Discussão]
Fala galera!

Tô tentando levantar aqui dentro da empresa alguns exemplos de empresas que possuem sites apenas para desenvolvedores (algo como o [Facebook Developers](http://developers.facebook.com), [Airbnb.io,](http://airbnb.io), tal..) com detalhes sobre APIs, por exemplo.

Também tô procurando se essas mesmas empresas (ou outras) possuem sites direcionados à comunidade mas não apenas para desenvolvedores, sites que contenham qualquer tipo de conteúdo que fomentem a interação entre usuários\entusiastas ou conteúdos produzidos pelas próprias empresas, tipo podcasts, videos, sei lá.

Além disso, vocês acham interessante que as empresas pensem em fazer coisas desse tipo? A empresa de vocês faz algo assim?

**Comment by phhoffmann** [comment]
Acho que o título tava bem ruim, agora tá menos ruim mas ainda não acho que explica exatamente o que eu quis dizer. rs

**Comment by vicainelli** [comment]
Alguns que eu conheço:

- https://inside.getyourguide.com/
- https://open.buffer.com/
- https://atlassian.design/

**Comment by onlurking** [comment]
Cara, tem algumas empresas que publicam o playbook delas (é um documento explicando como é o dia-a-dia do trabalho, cultura, setup dos projetos), acho que o melhor exemplo é o [Playbook da Thoughtbot](https://thoughtbot.com/plahttps://www.carbondesignsystem.com/guidelines/spacing/ybook) e o da [Valve](https://steamcdn-a.akamaihd.net/apps/valve/Valve_NewEmployeeHandbook.pdf) que é hilário(mas é um documento, não um site).

Na mesma pegada de comunidade como você falou, a Thoughtbot também tem [podcast](http://giantrobots.fm/), [livros](https://books.thoughtbot.com/), uma [plataforma de cursos](https://thoughtbot.com/upcase) e um canal no [youtube com as tech talks](https://www.youtube.com/user/ThoughtbotVideo/playlists) deles abertos para a comunidade. Eles também tem um [script de setup](https://github.com/thoughtbot/laptop) para máquinas de desenvolvimento.

Em outra pegada, indo pro lado de documentação de design e produto, algumas empresas deixam aberto o design system delas(é um documento com os padrões de design e ás vezes implementação), como a:
- IBM com o [Carbon Design System](https://www.carbondesignsystem.com/guidelines/spacing/)
- Salesforce com o [Lightning Design System](https://www.lightningdesignsystem.com/design-tokens/#category-color)
- [Zeit](https://zeit.co/design/color)
- Shopify com o [Polaris](https://polaris.shopify.com/design/colors#section-guidelines)

Nesse [link](https://github.com/alexpate/awesome-design-systems) tem uma listona com várias empresas e seus design systems.

Agora na parte de padronização de código, um ótimo exemplo é a Airbnb com seu styleguide pra [Javascript](https://github.com/airbnb/javascript) e [CSS](https://github.com/airbnb/css).
